app/code/Magento/LoginAsCustomerGraphQl/Model/LoginAsCustomer/CreateCustomerToken.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\LoginAsCustomerGraphQl\Model\LoginAsCustomer;
+
+use Magento\Customer\Model\CustomerFactory;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Integration\Model\Oauth\TokenFactory;
+use Magento\Store\Model\StoreManagerInterface;
+use Exception;
+
+/**
+ * Create customer token from customer email
+ *
+ * Class CreateCustomerToken
+ */
+class CreateCustomerToken
+{
+    /**
+     * @var CustomerFactory
+     */
+    private $customerFactory;
+
+    /**
+     * @var TokenFactory
+     */
+    private $tokenModelFactory;
+
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
+    /**
+     * CreateCustomerToken constructor.
+     * @param StoreManagerInterface $storeManager
+     * @param TokenFactory $tokenModelFactory
+     * @param CustomerFactory $customerFactory
+     */
+    public function __construct(
+        StoreManagerInterface $storeManager,
+        TokenFactory $tokenModelFactory,
+        CustomerFactory $customerFactory
+    ) {
+        $this->storeManager = $storeManager;
+        $this->tokenModelFactory = $tokenModelFactory;
+        $this->customerFactory= $customerFactory;
+    }
+
+    /**
+     * Get admin user token
+     *
+     * @param string $email
+     * @return array
+     * @throws LocalizedException
+     */
+    public function execute(string $email): array
+    {
+        $websiteID = $this->storeManager->getStore()->getWebsiteId();
```

Comment:
I would not recommend using this approach since it might be inconsistent (there's a chance of getting wrong store ID). In GraphQL area we get information about the current store from the GraphQL execution context but not from the store manager. 

Please, take a look at the following example 

https://github.com/magento/magento2/blob/765f3804e919e5f06f4941eb4a227258befe2995/app/code/Magento/QuoteGraphQl/Model/Resolver/AddSimpleProductsToCart.php#L62

That's the correct and recommended way of getting the current store. You may pass the website id as an argument to this function from resolver.